utils/node_local_test.py

```diff
@@ -3158,7 +3163,7 @@ def test_alloc_fail(server, conf):
     destroy_container(conf, pool, container)
     return rc
 
-def main():
+def main(wf):
```

Comment:
(pylint-inconsistent-return-statements) Either all return statements in a function should return an expression, or none of them should.
(pylint-too-many-branches) Too many branches (27/25)